doc/lsst.ap.pipe/pipeline-tutorial.rst

```diff
@@ -104,11 +108,11 @@ something like
 
 .. code-block:: none
 
+   association.db   <--- the Prompt Products Database with DIAObjects
    repo/
```

Comment:
What I take away from this is that the database with whatever name the user chooses will be created in the same directory which contains `repo`, assuming there is not already a file there with that name. Is that right?

Reply:
No, it's created relative to the working directory. Is there some obvious other place to put it, given that spelling out a full rerun path kind of defeats the point of reruns?

Reply:
That sentence sounds too much like saying there's a default location. How about changing the example to `-c associator.level1_db.db_name=ppdb/association.db` and modifying the "after" directory layout to match? Will that make things unambiguous?

Reply:
Yes, that sounds good. I understand you want to avoid hinting at any kind of default. Thanks!